CHANGELOG.md

```diff
@@ -120,7 +236,7 @@ Older Logs can be found in `docs/RELEASE-NOTES-*`.
 
 - Scipy becomes optional dependency (for polar channel code construction)
 
-#### gr_modtool
+#### `gr_modtool`
```

Comment:
Are you deliberately retro-changing the changelog entries for previous releases?

Reply:
I'll change those back ... it was a thing with markdown not rendering correctly. More important to keep it the same as before.